policy/approval/approve.go

```diff
@@ -57,13 +57,24 @@ type RequestReview struct {
 
 func (opts *Options) GetMethods() *common.Methods {
 	methods := opts.Methods
+	defaultBool := true
 	if methods == nil {
 		methods = &common.Methods{
 			Comments: []string{
 				":+1:",
 				"👍",
 			},
-			GithubReview: true,
+			GithubReview: &defaultBool,
+		}
+	} else {
+		if len(methods.Comments) == 0 {
```

Comment:
I think you could simplify both the if and the else into one. Effectively, if `methods == nil`, then `methods.Comments` and `methods.GithubReview` will also be nil.

Reply:
I agree, will make the change.